vignettes/design-principles.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "Design Principles for {packagetemplate}"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Design Principles for {packagetemplate}}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This vignette outlines the design decisions that have been taken during the development of the {packagetemplate} R package, and provides some of the reasoning, and possible pros and cons of each decision.
```

Comment:
```suggestion
This vignette outlines the design decisions that have been taken during the development of the `{packagetemplate}` R package, and provides some of the reasoning, and possible pros and cons of each decision.
```

If we're going to use the `{pkg}` syntax, it needs to be surrounded by backticks to be properly identified by downlit.